tox.ini

```diff
@@ -0,0 +1,53 @@
+# Tox (http://tox.testrun.org/) is a tool for running tests
+# in multiple virtualenvs. This configuration file will run the
+# test suite on all supported python versions. To use it, "pip install tox"
+# and then run "tox" from this directory.
+
+[tox]
+skipsdist = true
+envlist =
+    py35-sqlite
+    flake8
+    docs
+    isort
+
```

Comment:
The tox config doesn't actually need to include both of them; thanks to tox's automatic defaults, `py34-sqlite` is a valid test environment, which can be used instead of the default (`py35-sqlite`). To speed up the, checks only one is included by default.